_locales/en/messages.json

```diff
@@ -2207,6 +2300,9 @@
 	"ghostery_basic": {
 		"message": "Basic"
 	},
+	"ghostery": {
+		"message": "Ghostery"
+	},
```

Comment:
I believe we don't localize our name, so we should be able to just hardcode the string. I think this applies to a few other strings - check with @TeresaLan for the full list.

Reply:
Cool cool, I went through a list that she sent me and made sure all of those strings aren't localized

Reply:
Great, thanks!